src/test/scala/mesosphere/marathon/core/instance/TestInstanceBuilder.scala

```diff
@@ -16,7 +16,7 @@ import scala.concurrent.duration._
 case class TestInstanceBuilder(instance: Instance, now: Timestamp = Timestamp.now()) {
 
   def addTaskLaunched(container: Option[MesosContainer] = None): TestInstanceBuilder =
-    addTaskWithBuilder().taskLaunched(container).build()
+    addTaskWithBuilder().taskLaunched(container).build().withInstanceCondition(condition = Condition.Running)
```

Comment:
Why didn't we need this before?

Reply:
I don't really know, but it doesn't make sense to have an instance in `Created` state where tasks are already launched.

Reply:
`.build()` will call `instanceBuilder.addTask` which will call `InstanceUpdater.updatedInstance` which will make sure the instance has the correct Condition as per the contained tasks.

The while test fixture _situation_ is … unfortunate, but I don't think you need `withInstanceCondition`.

Reply:
*IF* we need to do that, we should do it inside `taskLaunched`...

Reply:
btw. where did this surface? What tests started to fail and why? I wonder why it did not break with created...